src/commands/config.ts

```diff
@@ -109,7 +110,17 @@ export const configValidators = {
       `${value} is not supported yet, use 'gpt-4' or 'gpt-3.5-turbo' (default)`
     );
     return value;
+  },
+
+  [CONFIG_KEYS.prefix](value: any) {
+    validateConfig(
+      CONFIG_KEYS.prefix,
+      true,
```

Comment:
true is not validating anything here :) we should validate the config input with a function or expression which return Boolean

Reply:
@di-sukharev Do you have an idea what can we validate?

Reply:
you somehow need to confirm that the value is the exact format you expect, if your format is `^(*)` and not `/^(*)/` — make sure you don't have `/`s in the value, you may also do multiple validations like `!a && b && !c`